Sources/NSBundleExtensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NSBundleExtensions.swift
+//  BugShaker
+//
+//  Created by Jake Payton on 6/24/16.
+//  Copyright © 2016 Dan Trenz. All rights reserved.
+//
+
+import Foundation
+
+extension NSBundle {
+
+    var releaseVersion: String? {
+        return self.infoDictionary?["CFBundleShortVersionString"] as? String
+    }
+
+    var buildVersion: String? {
+        return self.infoDictionary?["CFBundleVersion"] as? String
+    }
+
+    var appVersion: String {
+        return "\(self.releaseVersion!).\(self.buildVersion!)"
```

Comment:
I generally try to avoid implicitly unwrapped optionals whenever possible since they will cause a bad error if they are ever `nil`.  

Would you consider making `appVersion` an optional String, as well? Or, return an empty string for `appVersion` if either `releaseVersion` or `buildVersion` are `nil`?